src/services/mcp/McpHub.ts

```diff
@@ -830,23 +839,53 @@ export class McpHub {
 			console.error(`Failed to parse timeout configuration for server ${serverName}: ${error}`)
 		}
 
-		const result = await connection.client.request(
-			{
-				method: "tools/call",
-				params: {
-					name: toolName,
-					arguments: toolArguments,
-				},
-			},
-			CallToolResultSchema,
-			{
-				timeout,
-			},
+		this.telemetryService.captureMcpToolCall(
```

Comment:
If we import telemetryService directly from the suggested path:

```suggestion
		telemetryService.captureMcpToolCall(
```